### DISCUSSION TITLE
Help with error: [-11414] Buffered point does not accept new events

### DISCUSSION THREAD

**Discussion by cmrncrick** [Hugovdberg/PIconnect#775] [Q&A] [answered]
To start, I am using PIConnect V 0.10.6 and Python V 3.11.5. 

I am trying to write to some tags and am encountering an error each time. The error is coming back as:

```
> [-11414] Buffered point does not accept new events
> at OSIsoft.AF.PI.PIException.ConvertAndThrowException(PIServer piServer, Exception ex, String message)
> at OSIsoft.AF.PI.PIPoint.UpdateValueInternalBufferIfPossible(Events mdaEvents, AFUpdateOption option)
> at OSIsoft.AF.PI.PIPoint.UpdateValue(AFValue value, AFUpdateOption option, AFBufferOption bufferOption)
```

I have written to tags before without any issue but I've encountered a batch that is throwing this error. I have compared them in PI SMT and haven't found any obvious differences between the tags that I can write to and the ones I can't. This is just some test code that I threw together while troubleshooting the issue for reference. 

```python
t_write = "SMACT.SWARF_BGHSE_INLET_TEMP.15M"
point = server.search(t_write)[0]
timestamp = end_calc
value = float(swarf_bh_15m_avg_inlet_temp)

try:
    point.update_value(value, timestamp, UpdateMode.NO_REPLACE,
                       BufferMode.BUFFER_IF_POSSIBLE)
except Exception as e:
    logger.error(f"***ERROR*** {e}")
```

**Comment by Hugovdberg** [comment, marked as answer]
Perhaps this post can help: https://pisquare.osisoft.com/s/question/0D51I00004UHgPNSA1/how-to-solve-error-code-11414-buffered-point-does-not-accept-new-events.
